changelog/next/features/3036--put-operator.md

```diff
@@ -0,0 +1,3 @@
+The `put` operator succeeeds the `extend` and `replace` operators. It updates
+values in-place when the assigned-to extractor exists in the input events, and
+ a new field with the assigned value ad the end otherwise.
```

Comment:
```suggestion
The `put` operator succeeeds the `extend` and `replace` operators. It updates
values in-place when the assigned-to extractor exists in the input events, and
adds a new field with the given value at the end otherwise.
```